data/en/qcas_0018.json

```diff
@@ -0,0 +1,739 @@
+{
+    "eq_id": "qcas",
+    "form_type": "0018",
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "019",
+    "title": "Quarterly Acquisitions and Disposals of Capital Assets Survey",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "navigation": {
```

Comment:
Do we need this? Does it default to False anyway?

Reply:
Was waiting on business to make their mind up.